### PR TITLE
Collapsable sidebar

### PR DESCRIPTION
**Pull request by Matt-Hill-scty**
Make left sidebar collapsable.

### CODE REVIEW

File: src/js/controllers/baseCtrl.js [hunk 1]
```diff
@@ -2,4 +2,14 @@ controllers.controller('BaseCtrl',[BaseCtrl_]);
```
Comment (andrewjcarl) [inline src/js/controllers/baseCtrl.js]: remove console.log.
also this should be in its own controller that just manages the sidebar state

File: src/js/directives/sidebar/sidebar.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ function mainSidebar() {
   return {
     restrict: 'E',
     replace: true,
-    templateUrl: 'html/sidebar.html'
+    templateUrl: 'html/sidebar.html',
+    controller: 'BaseCtrl as base'
```
Comment (andrewjcarl) [inline src/js/directives/sidebar/sidebar.js]: see comment above -- I would suggest this should be its own controller for just the sidebar to keep separation of concerns

File: src/html/sidebar.html [hunk 3]
```diff
@@ -12,20 +11,32 @@
...
-  <div class="item sidebar-header">My Queue</div>
-
-  <work-queue></work-queue>
+  <div ng-show="!base.collapsed" class="item sidebar-header">My Queue</div>
+  <work-queue ng-show="!base.collapsed"></work-queue>
 
-  <a href="" class="item bottom">
-    <i class="fa fa-angle-double-right"></i>
+  <a href="" class="item bottom" ng-click="base.collapseSidebar()">
+    <i class="fa fa-angle-double-right">
```
Comment (andrewjcarl) [inline src/html/sidebar.html]: icon should change when it is closed vs. open

File: src/js/directives/sidebar/sidebar.js (hunk 2 above)
Comment (Matt-Hill-scty) [inline src/js/directives/sidebar/sidebar.js]: fixed

File: src/js/controllers/baseCtrl.js (hunk 1 above)
Comment (Matt-Hill-scty) [inline src/js/controllers/baseCtrl.js]: fixed

**Comment by Matt-Hill-scty** [comment]
all comments have been addressed.

**Pull request merged**